### ISSUE TITLE
JOSS REVIEW: Documentation

### ISSUE THREAD

**Issue by CFGrote** [Bios4Biol/intronSeeker#3]
[JOSS Review Issue](https://github.com/openjournals/joss-reviews/issues/6272)

The documentation is very detailed. Nevertheless I have some recommendations and comments:

1.) The readability could benefit from checking and fixing some language and typo issues such as:

- 
> In order to get the correct protein sequence introns have to be removed, Retained...

-> In order to get the correct protein sequence, introns have to be removed. Retained ...
What is meant by "Retained"? Are they retained on purpose or by accident? Does this mean that introns are detected and removed in some upstream data processing step but some may have been overlooked?

- The figure following "...corresponding assembled contigs." contains "ARNm", should read "mRNA".

- Open Reading Frame or open reading frame (only one should be used, maybe define and use  ORF)

2.) The "How it works" section shows a quite complex flowchart but only limited information is given in the text. The interested reader will wonder where to get more detailed information on the various elements in the flowchart. What's the meaning of dashed and solid lines (connectors)? What really is the input data for intronSeeker. Is it raw RNASeq reads? Mapped reads? What is the role of simulated data, is it used for validating the method or does intron detection involve a simulation step in any case?

3.) The readme mentions a "How to use" file in doc/ directory, but there is no such file.

4.) Third party tools:
Is it really required (useful?) to wrap aligners such a star and hisat? I'd assume any potential user will want to run their favourite aligner and customize the respective alignment parameters to their very individual needs. 

5.) Grinder file
the examples are hardly readable, please insert proper linebreaks

6.) API
As far as I see, there is no API documentation, but maybe there is no API and intronSeeker is rather a collection of CLI tools and wrappers?

7.) Automated tests
No automated tests as far as I can tell.

**Comment by Bios4Biol** [comment]
Thank you for your comments and suggestions for improvement.

1.) They are retained by "accident" meaning due to non mature transcripts present in the cell. These transcripts have not undergone complete splicing and therefore still harbor introns. The upstream step is not data processing but a biological process called splicing. 

2.) I'm working on point 2.

4.) The aligners we tested were the only two capable of handling alignments on contigs with and without the intron. We wrapped them to simplify their launch for users.

6.)  intronSeeker is a collection of CLI tools and wrappers.

7.) Indeed, automated tests have been removed.
